java/src/main/java/com/ibm/plugin/rules/detection/bc/BouncyCastleInfoMap.java

```diff
@@ -129,6 +132,7 @@ public Set<Entry<String, Info>> entrySet() {
      *
      * @param key - Key of the information map, typically the cryptographic class
      * @return The correct name of the cryptographic asset
+     * @deprecated use the key of the map instead
```

Comment:
Add the `@Deprected` annotation similar to [here](https://github.com/IBM/sonar-cryptography/blob/main/engine/src/main/java/com/ibm/engine/model/context/CipherContext.java#L61-L70)